aws/eks-helm/metrics-server/examples/main.tf

```diff
@@ -0,0 +1,7 @@
+module "example" {
+  source = "../"
+
+  helm_release_name  = "metrics-server"
+  namespace          = "kube-system"
+  helm_chart_version = "3.12"
```

Comment:
should `3.12.2` ?

Reply:
I forgot the example, I updated it, thank you.